src/sentry/plugins/sentry_mail/models.py

```diff
@@ -54,8 +54,9 @@ def _subject_prefix(self):
             return self.subject_prefix
         return options.get('mail.subject-prefix')
 
-    def _build_message(self, project, subject, template=None, html_template=None, body=None,
-                   reference=None, reply_reference=None, headers=None, context=None, send_to=None):
+    def _build_message(self, project, subject, template=None, html_template=None,
+                   body=None, reference=None, reply_reference=None, headers=None,
```

Comment:
We should provide a default string here for `type` to `other` or `generic`. Something not `None`, since I assume the `None` will end up getting repr'd to a type of `None`.